README.md

```diff
@@ -1,3 +1,15 @@
 # Download service
 
-Backend server for [https://github.com/cta-epfl/ctadata](https://github.com/cta-epfl/ctadata).
\ No newline at end of file
+Backend server for [https://github.com/cta-epfl/ctadata](https://github.com/cta-epfl/ctadata).
+
+## Installation
+
+For `production` use to install dependencies just run :
+```
+poetry install
+```
+
+For `test` purpose without jupyterhub :
+```
+poetry install --with dev --without jupyterhub
+```
```

Comment:
```suggestion
```

Note, however, that in this mode the service will not check authorization and will by default fail to respond to any request for restricted data. However, if `CTADS_DISABLE_ALL_AUTH` application config is set, the service will authorize all requests: this way this mode can also be used for for testing. 
```